### ISSUE TITLE
Load error with @mtkbuild

### ISSUE THREAD

**Issue by finmod** [bradcarman/ModelingToolkitWebinar#1]
@bradcarman Excellent webminar. You should indicate there that the demo.jl file is here.

I was trying to execute locally the demo.jl file. At:
`#equations(sys)

#full_equations(sys)

@mtkbuild sys = System()

#prob = ODEProblem(sys, [], (0, 0.1), [])
`

I get a LoadError : UndefVarError: `@mtkbuild` not defined

**Comment by bradcarman** [comment]
Hi @finmod, really glad you like the webinar!  

For your problem I can only suspect that the version of ModelingToolkit.jl you are using is old.  I was using version `ModelingToolkit v8.73.2`.  @mtkbuild is a fairly new addition to ModelingToolkit.jl

**Comment by finmod** [comment]
Correct. I have updated to ModelingToolkit@8.73.4 but the problem subsists. There seems to be a lot of compat problems with the latest versions of MTK

**Comment by finmod** [comment]
This issue has been solved in this latest version with Julia v1.10.0 and MTK v8.75.0. 

Instead, there is bug in the display of @mtkbuild in this webminar as: 
![image](https://github.com/bradcarman/ModelingToolkitWebinar/assets/12710434/9efa981a-1fec-425e-9d92-351dd470722a)

It does not exist in the RC Circuit tutorial for the same @mtkbuild command.
